giving-a-presentation.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Giving a Presentation
+sidebar_link: false
+---
+
+Boston Python's presentation events are a place for members of our community
+to share what they've learned about making python programs. Giving a
```

Comment:
```suggestion
to share what they've learned about making Python programs. Giving a
```

Reply:
This page doesn't push back on presentations motivated by self-interest, or pitches.  Is there a way we can do that gently?

Reply:
I was wondering about that. I haven't been able to come up with language that I liked for that. When I talk to people in person about this sort of proposal, I'm usually able to explain why this sort of talk isn't a good fit, and to redirect them towards something that would be a better fit. To that end, we would try something like this:

The best I can come up with, based on my in-person conversations is something like
```
### I have a really cool project that I think people could use. Can I give a talk about that?

We generally don't find that pitches for personal projects are a good fit our audience. Is 
there something you've learned from making this project that you think would be interesting to python
users? If so, maybe you can make a presentation out of that. Reach out to an organizer and 
let's talk about it!
```

However I think it might be better to have this conversation in person, so we can gauge whether the message has actually landed, or if they're just reworking their pitch to get past this filter.

Reply:
I can live with doing this filtering in in-person conversations.